app/routes.js

```diff
@@ -1,5 +1,9 @@
 const { healthCheck } = require('./controllers/healthCheck');
+const { newUser } = require('./controllers/user');
+const { validateSchemaAndFail } = require('./middlewares/params_validators');
+const { createUserSchema } = require('./schemas/users');
 
 exports.init = app => {
   app.get('/health', healthCheck);
+  app.post('/new-user', validateSchemaAndFail(createUserSchema), newUser);
```

Comment:
This endpoint is not using the HTTP rest standard, please read, please search about that and if you have any questions you always can type me by slack